packages/fxa-payments-server/src/utils.js

```diff
@@ -0,0 +1,74 @@
+import { ActionType as PromiseActionType } from 'redux-promise-middleware';
+const { assign } = Object;
+
+export const parseParams = params => params
+  .substr(1)
+  .split('&')
+  .reduce((acc, curr) => {
+    const parts = curr.split('=').map(decodeURIComponent);
+    acc[parts[0]] = parts[1];
+    return acc;
+  }, {});
+
+export const mapToObject = (list, mapFn) => {
```

Comment:
I suppose this could also be:

```js
export const mapToObject = (list, mapFn) => Object.fromEntries(list.map(item => [item, mapFn(item)]));
```

Reply:
I think `Object.fromEntries` is very new - would be nice to use if we roll in a polyfill maybe via babel?